README.md

```diff
@@ -1,108 +1,129 @@
-# community.sap_launchpad Ansible Collection ![Ansible Lint](https://github.com/sap-linuxlab/community.sap_launchpad/actions/workflows/ansible-lint.yml/badge.svg?branch=main)
+# community.sap_launchpad Ansible Collection
 
-This Ansible Collection executes basic SAP.com Support operations tasks.
+![Ansible Lint](https://github.com/sap-linuxlab/community.sap_launchpad/actions/workflows/ansible-lint.yml/badge.svg?branch=main)
 
-## Functionality
+## Description
 
-This Ansible Collection executes basic SAP.com Support operations tasks, including:
```

Comment:
Original statement is more accurate, because of Ansible Modules `license_keys` and `systems_info`